vllm/worker/model_runner.py

```diff
@@ -887,14 +887,7 @@ def capture_model(self, kv_caches: List[torch.Tensor]) -> None:
         logger.info("Graph capturing finished in %.0f secs.", elapsed_time)
 
     def __del__(self) -> None:
-        # Delete the CUDA graphs before deleting the pynccl communicator.
-        # NOTE(woosuk): This is necessary because otherwise deadlocks can
-        # happen.
-        # FIXME(woosuk): This is a bit hacky. Find a more robust solution.
-        # TODO(youkaichao): when we get enough user feedback that pynccl is
-        # more stable than cupy, we can remove this, e.g. in v0.4.1.
         self.graph_runners.clear()
```

Comment:
Actually, I think you can delete this as well.

Reply:
Okay, then I can delete the whole function.

Reply:
fixed in [b224913](https://github.com/vllm-project/vllm/pull/4866/commits/b224913830ea4872af2cee97f3402abce2726f46)